libs/models/player.ts

```diff
@@ -0,0 +1,19 @@
+interface TimeLog {
+  length: number; //typed text length.
+  distance: number; //time distance between initial point and current timestamp.
+}
+
+export interface Player {
+  id: number;
+  name: string;
+  userIcon: string;
+  lobbyId: number;
+}
+
+export interface RacePlayer extends Player {
```

Comment:
why do we extend the type here?
Do we expect `Player` to be used in some other places?